.gitlab-ci.yml

```
@@ -51,12 +51,10 @@ integration-testing:
 
 release-runner-image:
   stage: release
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:20.10.13
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/images/mirror/crane:latest
```

Should be now - you might have already fixed it
```suggestion
  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/images/mirror/crane:debug
```